feature-libs/organization/administration/styles/_card.scss

```diff
@@ -93,17 +96,35 @@
     margin: 40px;
     background-color: var(--cx-color-inverse);
     border: solid 1px var(--cx-color-light);
+    @include cx-highContrastTheme-light {
+      border: 1px solid rgba(0, 0, 0, 0.125);
+    }
+    @include cx-highContrastTheme-dark {
+      border: solid 1px var(--cx-color-text);
+    }
+    @include cx-highContrastTheme {
+      background-color: var(--cx-color-background);
+    }
 
     &.link-list {
       display: flex;
       flex-direction: column;
+      @include cx-highContrastTheme {
+        background-color: var(--cx-color-background);
+      }
       a {
         padding: 25px 30px;
         justify-content: flex-start;
         min-height: 74px;
 
         &:not(:last-child) {
           border-bottom: solid 1px var(--cx-color-light);
+          @include cx-highContrastTheme-light {
+            border-bottom: solid 1px rgba(0, 0, 0, 0.125);
```

Comment:
`query:` maybe we should consider moving `rgba(0, 0, 0, 0.125)` to a variable?

Reply:
Currently, we don't have a global common variable for the border color, unless we decide to introduce a local one.